### PR TITLE
Shopper view stores

### PR DESCRIPTION
**Pull request by Zain4Rizvi**
Implemented Shopper View Store As Fragment

### CODE REVIEW

File: app/src/main/java/com/example/b07group7project/GetStoreImplementation.java
Comment (Zain4Rizvi) [inline app/src/main/java/com/example/b07group7project/GetStoreImplementation.java:1]: Don't need to add this file but not sure how to remove individual files from pull request

**Comment by Zain4Rizvi** [comment]
Look over and make sure it dont override accidentally overwrite anything we need in main

File: app/src/main/java/com/example/b07group7project/GetStoreImplementation.java
Comment (aarondme) [inline app/src/main/java/com/example/b07group7project/GetStoreImplementation.java:1]: Replace ArrayList with List (if the method you use this for only requires List)

File: app/src/main/java/com/example/b07group7project/GetStoreInterface.java
Comment (aarondme) [inline app/src/main/java/com/example/b07group7project/GetStoreInterface.java:1]: Same as above, replace with List if that's convenient

File: app/src/main/java/com/example/b07group7project/Store.java
Comment (aarondme) [inline app/src/main/java/com/example/b07group7project/Store.java:1]: This works for now... talk to Seb about how to hook this up with the Image getter class

File: app/src/main/java/com/example/b07group7project/StoreRecyclerViewAdapter.java
Comment (aarondme) [inline app/src/main/java/com/example/b07group7project/StoreRecyclerViewAdapter.java:1]: This is setting the image as if it was stored on the project. It needs to use some image getter, talk to seb. Same kind of idea as what you did for abstracting over the database should be done over the image getter class for now

**Review by aarondme — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/b07group7project/StoreRecyclerViewAdapter.java
Comment (aarondme) [reply app/src/main/java/com/example/b07group7project/StoreRecyclerViewAdapter.java:1]: No need for the abstraction, just use it directly.

**Review by aarondme — APPROVED** [review]

**Pull request merged**